sections/clause_09a_bindings_http.adoc

```diff
@@ -35,6 +35,16 @@ Content for Create, Update, Replace or Execute requests is passed in the body of
 HTTP HEAD and OPTIONS requests are treated as normal GET requests, but don't return data.
 If a GET request with the same URL and headers would return an error, the HEAD or OPTION requests also returns this error.
 
+[[return-value]]
```

Comment:
return values are not binding specific, they can be applied to any binding, though a binding can modify or fix the return value.
We do need to fix the fact that _headers_ are mentioned...

Reply:
We rewrote this section during the telco, it should not be moved. Now causes a merge conflict...